counterparty-core/counterpartycore/lib/messages/dispenser.py

```diff
@@ -410,6 +410,30 @@ def compose(
     return (source, destination, data)
 
 
+def compose_dispense(db, source, destination, quantity, no_only_btc=False):
+    if not util.enabled("enable_dispense_tx"):
+        raise exceptions.ComposeError("dispense tx is not enabled")
+    dispensers = ledger.get_dispensers(db, address=destination)
+    if len(dispensers) == 0:
+        # simple BTC send
+        if no_only_btc:
```

Comment:
when is this condition true?

Reply:
when you call `dispense.compose()`

Reply:
refactored a little to make it cleaner..